src/content/scrolling.ts

```diff
@@ -115,35 +115,35 @@ const verticallyScrolling = new Map<Node, ScrollingData>()
  *  last duration milliseconds
  */
 export async function scroll(
-    x: number = 0,
-    y: number = 0,
+    xDistance : number = 0,
+    yDinstance : number = 0,
     e: Node,
     duration?: number,
-) {
+) : Promise<Boolean> {
```

Comment:
```suggestion
) : Promise<boolean> {
```

https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html :)